netty/src/main/java/io/grpc/netty/ClientTransportLifecycleManager.java

```diff
@@ -64,7 +64,6 @@ public boolean notifyShutdown(Status s) {
       return false;
     }
     shutdownStatus = s;
-    shutdownThrowable = s.asException();
```

Comment:
The point of this is to be when the channel is created, so you know "which channel is this." We can discuss other options, like printing `channel.authority()` instead of using a stacktrace (which is less reliable, but less costly).

Reply:
Oh right, my change is stupid as it doesn't capture the correct stacktrace. I can delete this commit and create a dedicated issue for picking a less costly way. Agree?

Reply:
Sounds good.

Reply:
Done

Reply:
See https://github.com/grpc/grpc-java/issues/10524